### PR TITLE
all: simplify import paths

### PR DESCRIPTION
**Pull request by sbinet**
- rename `package generate` into `package gen` to match import path
- simplify import path for `github.com/go-clang/gen/clang` as well

### CODE REVIEW

File: clang/cmd.go [hunk 1]
```diff
@@ -6,9 +6,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/go-clang/gen"
```
Comment (zimmski) [inline clang/cmd.go]: I am a bit cumbersome when it comes to imports. I prefer to group then in three groups:
- STD imports
- third party imports
- project imports

File: cmd/go-clang-gen/main.go [hunk 2]
```diff
@@ -5,12 +5,12 @@ import (
 	"os"
 	"strings"
 
-	generate "github.com/go-clang/gen"
-	generateclang "github.com/go-clang/gen/clang"
+	"github.com/go-clang/gen"
+	"github.com/go-clang/gen/clang"
```
Comment (zimmski) [inline cmd/go-clang-gen/main.go]: I would prefer to see an explicit package name `genclang` here to make it clear that it is `gen/clang` and not one of our `clang` packages that is imported here.

**Comment by zimmski** [comment]
Just to remarks. Otherwise LGTM

File: clang/cmd.go (hunk 1 above)
Comment (sbinet) [inline clang/cmd.go]: I had just let `goimports` do its thing but ok, fine, 3-tiers import it is.

**Comment by sbinet** [comment]
done.

**Pull request merged**